src/app/app.component.html

```diff
@@ -8,6 +8,7 @@ <h1>
 
 
 </div>
+  <legend class="row legend-style" style=" color: black;"> We are blessed with a baby Girl named Sia. We wanted to create a personalized message wall for her to revisit all the well wishes/love she has received while she was a tiny baby to remember. We'd love to get your blessings for Sia on all special occasions - Diwali, Holi , Christmas, Birthday's etc...</legend>
```

Comment:
i think this message will need to move in signin.html
else it would be displayed on all the pages

Reply:
Done